src/uu/cksum/src/cksum.rs

```diff
@@ -148,4 +187,11 @@ pub fn uu_app() -> Command {
                 .action(clap::ArgAction::Append)
                 .value_hint(clap::ValueHint::FilePath),
         )
+        .arg(
+            Arg::new(options::ALGORITHM)
+                .long(options::ALGORITHM)
+                .short('a')
+                .help("select the digest type to use. See DIGEST below")
```

Comment:
There's no text on `DIGEST` below 😉 You can use the `after_help` function from `clap` to add some text below the options.

Reply:
done